app/controllers/finance/bank_accounts_controller.rb

```diff
@@ -5,6 +5,14 @@ def index
     redirect_to finance_bank_account_transactions_url(@bank_accounts.first) if @bank_accounts.count == 1
   end
 
+  def assign_unlinked_transactions
+    @bank_account = BankAccount.find(params[:id])
```

Comment:
What about `params.require(:id)`?

Reply:
we don't do this anywhere else. i'd like to stick to existing style